apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/UpdateSearchRequest.tsx

```diff
@@ -4,9 +4,17 @@ import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
 import { useIntl } from "react-intl";
 
 import { Button, Heading } from "@gc-digital-talent/ui";
-import { Submit, TextArea } from "@gc-digital-talent/forms";
+import {
+  SelectFieldV2,
```

Comment:
Could we avoid using the `SelectFieldV2` if possible? It has many a11y problems and I'm not sure it serves any benefit here. A regular `Select` would be better imo.

Reply:
I think it is better too in this case, no null selection https://github.com/GCTC-NTGC/gc-digital-talent/pull/7210/commits/e1fb95146b7d81f434e0e5e32ae17ee3cda3a52d